WpfAppMedicalSystemsDraft/MainWindow.xaml.cs

```diff
@@ -40,7 +40,7 @@ class AppSettings
     /// </summary>
     public partial class MainWindow : Window
     {
-        public string AccountTypeEnum { get; private set; } = AccountType.NOT_LOGGED;
```

Comment:
Przywróć typ konta

Reply:
to znaczy ustawić na PACIENT?, bo u mnie jest PACIENT i nie wiem chyba sie jakieś zmiany nie zapisały

Reply:
![image](https://github.com/Medical-system-UMCS/medical-system/assets/47827537/db7bd13c-f3dd-4155-a717-0abc295f313b)

Reply:
Powinno być NOT_LOGGED